src/lib/Components/Bidding/Screens/ConfirmFirstTimeBid.tsx

```diff
@@ -1,312 +0,0 @@
-import { commitMutation, createFragmentContainer, graphql } from "react-relay"
-import styled from "styled-components/native"
-import stripe from "tipsi-stripe"
-
-import SwitchBoard from "lib/NativeModules/SwitchBoard"
-import { Schema, screenTrack, track } from "../../../utils/track"
-
-import { Flex } from "../Elements/Flex"
-import { Serif14, SerifItalic14, SerifSemibold14, SerifSemibold18 } from "../Elements/Typography"
-
-import { BiddingThemeProvider } from "../Components/BiddingThemeProvider"
-import { BidInfoRow } from "../Components/BidInfoRow"
-import { Button } from "../Components/Button"
-import { Checkbox } from "../Components/Checkbox"
-import { Container } from "../Components/Containers"
-import { Divider } from "../Components/Divider"
-import { Timer } from "../Components/Timer"
-import { Title } from "../Components/Title"
-import { Address, BidderPositionResult, PaymentCardTextFieldParams, StripeToken } from "../types"
-
-import { BidResultScreen } from "./BidResult"
-import { BillingAddress } from "./BillingAddress"
-import { bidderPositionMutation, ConfirmBidProps, queryForBidPosition } from "./ConfirmBid"
-import { CreditCardForm } from "./CreditCardForm"
-
-const Emission = NativeModules.Emission || {}
-
-stripe.setOptions({ publishableKey: Emission.stripePublishableKey })
-
-interface ConfirmBidState {
-  billingAddress?: Address
-  creditCardFormParams?: PaymentCardTextFieldParams
-  creditCardToken?: StripeToken
-  conditionsOfSaleChecked: boolean
-  isLoading: boolean
-}
-
-const MAX_POLL_ATTEMPTS = 20
-
-const creditCardMutation = graphql`
-  mutation ConfirmFirstTimeBidMutation($input: CreditCardInput!) {
-    createCreditCard(input: $input) {
-      credit_card {
-        id
-        brand
-        name
-        last_digits
-        expiration_month
-        expiration_year
-      }
-    }
-  }
-`
-
-@screenTrack({
-  context_screen: Schema.PageNames.BidFlowConfirmBidPage,
-  context_screen_owner_type: null,
-})
-export class ConfirmFirstTimeBid extends React.Component<ConfirmBidProps, ConfirmBidState> {
-  state = {
-    billingAddress: null,
-    creditCardToken: null,
-    creditCardFormParams: null,
-    conditionsOfSaleChecked: false,
-    isLoading: false,
-  }
-
-  private pollCount = 0
-
-  onPressConditionsOfSale = () => {
-    SwitchBoard.presentModalViewController(this, "/conditions-of-sale?present_modally=true")
-  }
-
-  showCreditCardForm() {
-    this.props.navigator.push({
-      component: CreditCardForm,
-      title: "",
-      passProps: {
-        onSubmit: this.onCreditCardAdded,
-        navigator: this.props.navigator,
-      },
-    })
-  }
-
-  showBillingAddressForm() {
-    this.props.navigator.push({
-      component: BillingAddress,
-      title: "",
-      passProps: {
-        onSubmit: this.onBillingAddressAdded,
-        billingAddress: this.state.billingAddress,
-        navigator: this.props.navigator,
-      },
-    })
-  }
-
-  canPlaceBid() {
```

Comment:
#minor looks like this didn't make it up to the consolidated component, but the logic would change slightly anyways  so I'll make this part of a follow-up PR.

Reply:
I've added this back but feel free to update as you need.